scipy/stats/tests/test_hypotests.py

```diff
@@ -159,3 +160,79 @@ def test_callable_cdf(self):
         r1 = cramervonmises(x, distributions.beta.cdf, args)
         r2 = cramervonmises(x, "beta", args)
         assert_equal((r1.statistic, r1.pvalue), (r2.statistic, r2.pvalue))
+
+
+class TestCvm_2samp(object):
+    def test_invalid_input(self):
+        x = np.arange(10).reshape((2, 5))
+        y = np.arange(5)
+        assert_raises(ValueError, cramervonmises_2samp, x, y)
+        assert_raises(ValueError, cramervonmises_2samp, y, x)
+        assert_raises(ValueError, cramervonmises_2samp, [], y)
+        assert_raises(ValueError, cramervonmises_2samp, y, [1])
+        assert_raises(ValueError, cramervonmises_2samp, y, y, 'wrong_method')
+
+    def test_list_input(self):
+        x = [2, 3, 4, 7, 6]
+        y = [0.2, 0.7, 12, 18]
+        r1 = cramervonmises_2samp(x, y)
+        r2 = cramervonmises_2samp(np.array(x), np.array(y))
+        assert_equal((r1.statistic, r1.pvalue), (r2.statistic, r2.pvalue))
+
+    def test_example_conover(self):
+        # Example 2 in Section 6.2 of W.J. Conover: Practical Nonparametric
+        # Statistics, 1971.
+        x = [7.6, 8.4, 8.6, 8.7, 9.3, 9.9, 10.1, 10.6, 11.2]
+        y = [5.2, 5.7, 5.9, 6.5, 6.8, 8.2, 9.1, 9.8, 10.8, 11.3, 11.5, 12.3, 12.5, 13.4, 14.6]
+        r = cramervonmises_2samp(x, y)
+        assert_almost_equal(r.statistic, 0.262, decimal=3)
+        assert_almost_equal(r.pvalue, 0.18, decimal=2)
+
+    def test_exact_pvalue(self):
+        # the exact values are taken from Anderson: On the distribution of the
+        # two-sample Cramer-von-Mises criterion, 1962.
+        # The values are taken from Table 2, 3 and 4
+        p1 = _pval_cvm_2samp_exact(710, 5, 6)
+        p2 = _pval_cvm_2samp_exact(1897, 7, 7)
+        p3 = _pval_cvm_2samp_exact(576, 4, 6)
```

Comment:
```suggestion
        p3 = _pval_cvm_2samp_exact(576, 4, 6)
        p4 = _pval_cvm_2samp_exact(1764, 6, 7)
```

Reply:
I added the test case and parametrized the test for better readability